concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/08 Register 3rd-Party Dependencies/01 JSZip Registration.md

```diff
@@ -1,4 +1,4 @@
-DevExtreme requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a>. Since JSZip v3.3.0, the library does not need to be registered. If you use an earlier version, register JSZip in the `tsconfig.json` file:
+Some of the DevExtreme UI components requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a>. Since JSZip v3.3.0, the library does not need to be registered. If you use an earlier version, register JSZip in the `tsconfig.json` file:
```

Comment:
In v21.1, we remove this entire article. Why not in v20.2?

Reply:
I've already made a cherry-pick from 21.1 to 20.2. I submitted a separate PR to 20.1 since in that release we still required JSZip for the PivotGrid export.